src/Test/Plutip/LocalCluster.hs

```diff
@@ -92,7 +92,9 @@ withConfiguredCluster conf name testCases =
         traverse
           (traverse addSomeWallet . fmap twInitDistribuition . unTestWallets . fst)
           testCases
-      waitSeconds 2 -- wait for transactions to submit
+      -- had to bump waiting period here coz of chain-index slowdown,
+      -- see https://github.com/mlabs-haskell/plutip/issues/120
+      waitSeconds 5 -- wait for transactions to submit
```

Comment:
Consider extracting into some kind of Config?

Reply:
Made an issue for that: #125